app/models/blog.rb

```diff
@@ -10,11 +10,15 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE :term OR content LIKE :term', {term: "%#{sanitize_sql_like(term)}%"}) unless term.nil?
```

Comment:
```suggestion
    where('title LIKE :term OR content LIKE :term', term: "%#{sanitize_sql_like(term.to_s)}%")
```

Rubyは引数の最後のハッシュは`{}`は省略できるので、省略することが多いですね。
あと、`sanitize_sql_like`にnilを渡せないなら、`to_s`で文字列に変換する方がRubyらしいかなと思います。

また、termがnilだったらこのscopeの戻り値がnilになり、scopeの挙動を推測しにくくなる点も問題です。

Reply:
修正しました。